src/Databases/DatabaseFactory.cpp

```diff
@@ -211,14 +211,9 @@ DatabasePtr DatabaseFactory::getImpl(const ASTCreateQuery & create, const String
             if (engine_define->settings)
                 materialize_mode_settings->loadFromQuery(*engine_define);
 
-            if (create.uuid == UUIDHelpers::Nil)
```

Comment:
If `create.uuid` is Nil, we should throw exception which explains that it's not supported anymore and how to fix it and make server start (remove metadata file and database dirs)

Reply:
Done.